atef/bin/config.py

```diff
@@ -18,6 +19,39 @@ def build_arg_parser(argparser=None):
     if argparser is None:
         argparser = argparse.ArgumentParser()
 
+    # Arguments that need to be passed through to Qt
+    qt_args = {
+        '--qmljsdebugger': 1,
+        '--reverse': '?',
+        '--stylesheet': 1,
+        '--widgetcount': '?',
+        '--platform': 1,
+        '--platformpluginpath': 1,
+        '--platformtheme': 1,
+        '--plugin': 1,
+        '--qwindowgeometry': 1,
+        '--qwindowicon': 1,
+        '--qwindowtitle': 1,
+        '--session': 1,
+        '--display': 1,
+        '--geometry': 1
+    }
+
+    for name in qt_args:
+        argparser.add_argument(
+            name,
+            type=str,
+            nargs=qt_args[name]
+        )
+
+    argparser.add_argument(
+        '--style',
+        type=str,
+        choices=QStyleFactory.keys(),
+        default='fusion' if sys.platform == 'darwin' else None,
+        help='Qt style to use for the application'
```

Comment:
Maybe we can include a link to the qt docs as help text either here or somewhere more suitable, to explain all the args when someone calls --help
https://doc.qt.io/qt-5/qguiapplication.html#supported-command-line-options

Reply:
I guess we could also just copy paste the text in but that sounds annoying

Reply:
I ended up linking to the docs in the argparse description string

Reply:
Can you add the second link as well?
Oddly enough, the args here are split between those two different docs pages with no linkage between them...